config/environments/production.rb

```diff
@@ -41,7 +41,7 @@
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
-  config.active_storage.service = :production
+  config.active_storage.service = :local
```

Comment:
This is the default value. See example on PDC Discovery: https://github.com/pulibrary/pdc_discovery/blob/main/config/environments/production.rb#L44